### ISSUE TITLE
[bug] No intersection of coverage error and empirical quantile on conformity score

### ISSUE THREAD

**Issue by leoniewgnr** [ourownstory/neural_prophet#1245]
**Describe the bug**
When showing the figure of `plot_nonconformity_scores` in the `test test_split_conformal_prediction` using the cqr method, the 1-α coverage error (green line) and the empirical quantile q1 (red line), don't meet on the conformity score (blue line). 

**To Reproduce**
Run the following code and add a `fig.show()` to `uncertainty.py` to see the plot of `fig = plot_nonconformity_scores(self.noncon_scores, self.alpha, self.q_hats[0], method)`
```
def test_split_conformal_prediction():
    log.info("testing: Naive Split Conformal Prediction Air Travel")
    df = pd.read_csv(AIR_FILE)
    m = NeuralProphet(
        seasonality_mode="multiplicative",
        loss_func="MSE",
        quantiles=[0.05, 0.95],
        epochs=EPOCHS,
        batch_size=BATCH_SIZE,
        learning_rate=LR,
    )

    train_df, test_df = m.split_df(df, freq="MS", valid_p=0.2)
    train_df, cal_df = m.split_df(train_df, freq="MS", valid_p=0.15)
    metrics_df = m.fit(train_df, freq="MS")

    alpha = 0.1
    decompose = False
    plotting_backend = "default"
    for method in ["cqr"]: #["naive", "cqr"]:  # Naive and CQR SCP methods
        future = m.make_future_dataframe(
            test_df,
            periods=50,
            n_historic_predictions=len(test_df),
        )
        forecast = m.conformal_predict(
            future,
            calibration_df=cal_df,
            alpha=alpha,
            method=method,
            plotting_backend=plotting_backend,
            decompose=decompose,
        )
        eval_df = uncertainty_evaluate(forecast)
```
**Expected behavior**
The intersection of the green and red line, should lie on the blue line. 

**Environement **
 - Python environment: Python 3.9, in standalone venv with no other packages
 - Newest NeuralProphet version cloned from Github
![image](https://user-images.githubusercontent.com/42536262/229982147-fe7e7166-ccb3-49ee-a6d0-7b1abd2851c1.png)

**Comment by leoniewgnr** [comment]
@Kevin-Chen0 did you close that one with https://github.com/ourownstory/neural_prophet/pull/1248?

**Comment by leoniewgnr** [comment]
closed with #1248
